$NetBSD: patch-aa,v 1.3 2011/05/16 13:18:59 taca Exp $

Avoid depends on gd extension in package.xml since it is covered by
pkgsrc's denedency.

--- ../package.xml.orig	1970-01-01 09:13:08.000000000 +0000
+++ ../package.xml
@@ -51,9 +51,6 @@ Relicensing under LGPL 2.1.
    <pearinstaller>
     <min>1.4.0b1</min>
    </pearinstaller>
-   <extension>
-    <name>gd</name>
-   </extension>
   </required>
  </dependencies>
  <phprelease />
